### PR TITLE
feature/side-cabin

### PR DESCRIPTION
**Pull request by JoepEilander**

### CODE REVIEW

File: Assets/Art/2D/Textures/Train/T_TrainSideCabin.png
Comment (lucasvdiepen) [inline Assets/Art/2D/Textures/Train/T_TrainSideCabin.png:1]: Is it really necessary to increase the texture size?

**Review by TryFox — COMMENTED** [review]
All files have been approved. It would be nice if the doors would not ben the same colour as the walls, so they stand out more

**Review by TryFox — APPROVED** [review]
Approved changes

**Review by lucasvdiepen — CHANGES_REQUESTED** [review]
Colliders are removed from the train side cabin

**Review by lucasvdiepen — APPROVED** [review]

**Review by lucasvdiepen — APPROVED** [review]

**Review by Feetjexx — APPROVED** [review]
Approved

**Pull request merged**